.circleci/config.yml

```diff
@@ -46,6 +45,9 @@ jobs:
             # Update DNSlink
             if [ "$CIRCLE_BRANCH" == "main" ] ; then
               dnslink-dnsimple -d $DOMAIN -r _dnslink -l /ipfs/$hash
+              export CLUSTER_HOST=/dnsaddr/ipfs-websites.collab.ipfscluster.io
+              pin-to-cluster.sh "$DOMAIN" /tmp/workspace/$BUILD_DIR
+              echo "Website added to collaborative cluster"
```

Comment:
This will break the live dnslink because `$hash` is no longer defined. What was wrong with the previous script line and using docker to set `CLUSTER_HOST`?

Reply:
ah yes, I missed that in the folded code view. Now I see what you're doing here :)